client/connect-account-page/index.tsx

```diff
@@ -122,9 +130,23 @@ const ConnectAccountPage: React.FC = () => {
 			return handleLocationCheck();
 		}
 
+		trackModeSelected( 'live' );
+
 		window.location.href = connectUrl;
 	};
 
+	const handleEnableSandboxMode = async () => {
```

Comment:
Nitpick: It looks like there are a few things that happen in the `handleSetup` function like recording the `connect_account_clicked` event and enabling incentives, that we probably would also want to do here. 

One suggestion would be to instead of having two functions, make `handleSetup` accept a mode parameter (`live` or `test`) and it could track the mode selected and add the URL query args only in the case of a test account, but otherwise behave the same. WDYT?

Reply:
It was my first idea, but then I noticed that function is doing two jobs by accepting a flag argument and remembered that rule 
“Flag Arguments are ugly” It is a bad practice to send arguments to decision. It goes against the “one function, one thing” principle. https://ardalis.com/are-boolean-flags-on-methods-a-code-smell/

WDYT?